
[44m[1mFeature:[22m [3mRetry[23m[49m

  [44m[1mScenario:[22m [3mTest cases that pass aren't retried[23m[49m [90m# samples/retry/retry.feature:8[39m
    [32m[1mGiven [22m[3ma step that always passes[23m[39m             [90m# samples/retry/retry.ts:3[39m

  [44m[1mScenario:[22m [3mTest cases that fail are retried if within the --retry limit[23m[49m [90m# samples/retry/retry.feature:11[39m
    [31m[1mGiven [22m[3ma step that passes the second time[23m[39m                             [90m# samples/retry/retry.ts:8[39m
        [31mError: Exception in step[39m
        [31msamples/retry/retry.feature:12[39m

  [44m[1mScenario:[22m [3mTest cases that fail are retried if within the --retry limit[23m[49m [90m# samples/retry/retry.feature:11[39m
    [32m[1mGiven [22m[3ma step that passes the second time[23m[39m                             [90m# samples/retry/retry.ts:8[39m

  [44m[1mScenario:[22m [3mTest cases that fail will continue to retry up to the --retry limit[23m[49m [90m# samples/retry/retry.feature:14[39m
    [31m[1mGiven [22m[3ma step that passes the third time[23m[39m                                     [90m# samples/retry/retry.ts:16[39m
        [31mError: Exception in step[39m
        [31msamples/retry/retry.feature:15[39m

  [44m[1mScenario:[22m [3mTest cases that fail will continue to retry up to the --retry limit[23m[49m [90m# samples/retry/retry.feature:14[39m
    [31m[1mGiven [22m[3ma step that passes the third time[23m[39m                                     [90m# samples/retry/retry.ts:16[39m
        [31mError: Exception in step[39m
        [31msamples/retry/retry.feature:15[39m

  [44m[1mScenario:[22m [3mTest cases that fail will continue to retry up to the --retry limit[23m[49m [90m# samples/retry/retry.feature:14[39m
    [32m[1mGiven [22m[3ma step that passes the third time[23m[39m                                     [90m# samples/retry/retry.ts:16[39m

  [44m[1mScenario:[22m [3mTest cases won't retry after failing more than the --retry limit[23m[49m [90m# samples/retry/retry.feature:17[39m
    [31m[1mGiven [22m[3ma step that always fails[23m[39m                                           [90m# samples/retry/retry.ts:23[39m
        [31mError: Exception in step[39m
        [31msamples/retry/retry.feature:18[39m

  [44m[1mScenario:[22m [3mTest cases won't retry after failing more than the --retry limit[23m[49m [90m# samples/retry/retry.feature:17[39m
    [31m[1mGiven [22m[3ma step that always fails[23m[39m                                           [90m# samples/retry/retry.ts:23[39m
        [31mError: Exception in step[39m
        [31msamples/retry/retry.feature:18[39m

  [44m[1mScenario:[22m [3mTest cases won't retry after failing more than the --retry limit[23m[49m [90m# samples/retry/retry.feature:17[39m
    [31m[1mGiven [22m[3ma step that always fails[23m[39m                                           [90m# samples/retry/retry.ts:23[39m
        [31mError: Exception in step[39m
        [31msamples/retry/retry.feature:18[39m

  [44m[1mScenario:[22m [3mTest cases won't retry when the status is UNDEFINED[23m[49m [90m# samples/retry/retry.feature:20[39m
    [33m[1mGiven [22m[3ma non-existent step[23m[39m
